src/app/modules/ix-table2/components/ix-table-body/cells/ix-cell-text/ix-cell-text.component.html

```diff
@@ -1 +1,3 @@
-{{ value }}
+<span [matTooltip]="value?.toString()">
```

Comment:
I noticed that you've assuming that `value` will always have `toString()` method. I'd suggested to allow a broader range of values, and include even those without `toString()` method, and use `[matTooltip]="value"` until there's a reason doing otherwise

Reply:
![image](https://github.com/truenas/webui/assets/22980553/40fc393b-6e91-4076-9a2a-d6499ce1111a)